### PR TITLE
Update to MPV to use remote resources

### PR DESCRIPTION
**Pull request by echeipesh**
This may be impacted by changes to `TMSServer` which are not yet merged

### CODE REVIEW

**Comment by echeipesh** [comment]
Added the Elevation based workbook which will be impacted by the pyramid as well. It revealed https://github.com/locationtech-labs/geopyspark/issues/353 issue which may or may not exist here.

File: notebooks/mvp.ipynb [hunk 1]
```diff
@@ -26,13 +14,14 @@
...
-    "from geopyspark.geotrellis.color import get_breaks, ColorMap\n",
+    "from geopyspark.geotrellis.color import ColorMap\n",
     "from geopyspark.geotrellis.constants import *\n",
     "import json\n",
     "import shapely\n",
     "from shapely.geometry import shape, asShape, MultiPoint, MultiLineString\n",
     "\n",
-    "conf = geopyspark_conf(\"local[*]\", \"MVP\")\n",
+    "conf = geopyspark_conf(\"yarn-client\", \"MVP\")\n",
+    "conf.set(\"spark.hadoop.yarn.timeline-service.enabled\", False)\n",
```
Comment (echeipesh) [inline notebooks/mvp.ipynb]: @lossyrob This takes care of class def

**Comment by jamesmcclain** [comment]
These changes were added as part of geodocker-jupyter-geopyspark#17

**Comment by echeipesh** [comment]
Superseded by https://github.com/geodocker/geodocker-jupyter-geopyspark/pull/48

**Pull request closed without merging**